static/admin/config.yml

```diff
@@ -780,6 +780,331 @@ collections:
               default: "/earn/staker",
             }
           - { label: "Title", name: "title", widget: "string" }
+          - {
+              label: "Staker Info Section",
+              name: "stakerInfo",
+              widget: "object",
+              collapsed: true,
+              fields:
+                [
+                  {
+                    label: "Row Reverse",
+                    name: "rowReverse",
+                    widget: boolean,
+                    required: false,
+                    default: false,
+                  },
+                  {
+                    label: "Title",
+                    name: "title",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "markdown",
```

Comment:
If this field is a `markdown` and we want to inject as HTML we need to update our types at `gatsby-node`. Please see #22 as reference.

Reply:
I think it's fine to use string here in that case. If designers request a need to use markdown in this section we can edit later.